plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/NodeCleanupBeforeCompileHook.java

```diff
@@ -47,9 +47,6 @@ public void validate(Map<String, Object> toCheck) {
     private void compile(MavenRunner.Config mavenConfig, File path) throws PomExecutionException, IOException {
         System.out.println("Calling removeNodeFolders");
         removeNodeFolders(path);
-        System.out.println("Compile plugin log in " + path);
-        File compilePomLogfile = new File(path + "/compilePluginLog.log");
-        runner.run(mavenConfig, path, compilePomLogfile, "clean", "process-test-classes", "-Dmaven.javadoc.skip");
```

Comment:
Yeah, but that was before https://github.com/jenkinsci/plugin-compat-tester/pull/383/commits/6f446ae8ae7bea08236e1b22de28b5fc345ce026. I confirmed that previously it got compiled twice. I will check again though. I'm not really sure why this hook even exists, because I don't think these are the only two plugins that use `frontend-maven-plugin`, and a quick look at https://github.com/jenkinsci/plugin-compat-tester/pull/247 did not clarify the underlying issue.

Reply:
(I am retesting `sse-gateway` now, but NPM is extremely slow, maybe because `sse-gateway` is using very old versions of `node` and `npm` and so everything is running via Rosetta on my machine)

Reply:
I confirmed that `sse-gateway` still works.